aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -29,6 +29,15 @@ module aptos_framework::aggregator {
         limit: u128,
     }
 
+    /// Represents snapshot of an aggregator at a particular point in time.
```

Comment:
Maybe let's add a new module `aggregator_v2` already? All these beautiful new things can go there.

Reply:
To distinguish between Aggregator V1 and Aggregator V2, I saw that you are using `enum { Legacy, Ephemeral }` now. After adding Aggregator V2, are we going to remove the codebase for Aggregator V1? Or do we have to retain the code for compatibility reasons?

Reply:
We have to keep it at least for some time because supply is Aggregator V1